### PR TITLE
Adapt the 'rpm-ostree' test suite to continue on failure

### PR DESCRIPTION
**Pull request by miabbott**
This adapts the `rpm-ostree` test suite to the continue on failure model.  As part of the work, I made some slight changes to the 'compose' portion of the test suite to make it a bit more resilient.

Additionally, I snuck in a change to the `.gitignore` file to ignore the `.log` files that are being generated.

Partially addresses #375

### CODE REVIEW

**Comment by miabbott** [comment]
bot, retest this please

File: tests/rpm-ostree/deploy_version.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+# vim: set ft=ansible:
+#
+
+# https://bugzilla.redhat.com/show_bug.cgi?id=1571264
+- name: Run ostree admin status as a quick sanity check
+  command: ostree admin status
+  tags:
+    - ostree_admin_status
```
[the remark this answers is not in the document]
Comment (miabbott) [reply tests/rpm-ostree/deploy_version.yml]: Probably a good candidate for the sanity test

**Comment by miabbott** [comment]
Oof...that was a lazy commit.  Trying to use a task among the `roles:`  :disappointed: 

Working on a fix...

**Comment by mike-nguyen** [comment]
LGTM

**Pull request merged**